### PR TITLE
Pyolite: Fix Matplotlib

### LINKED ISSUE #161: Pyolite: Matplotlib not working anymore
I think due to the Comm support PRs.

matplotlib

### PR DESCRIPTION
**Pull request by martinRenou**
## References

Fix #161

This is temporary. At some point we should not monkey patch IPython...

### CODE REVIEW

File: packages/pyolite-kernel/py/pyolite/pyolite/__init__.py [hunk 1]
```diff
@@ -36,7 +36,11 @@ class InteractiveShellMock:
...
 ip.InteractiveShell = InteractiveShellMock
 
+pt = types.ModuleType("pylabtools")
+pt.backend2gui = {}
+
 sys.modules["IPython.display"] = pyolite.display
 sys.modules["IPython"] = ip
 sys.modules["IPython.core.getipython"] = ip
 sys.modules["IPython.core.interactiveshell"] = ip
+sys.modules["IPython.core.pylabtools"] = pt
```
Comment (martinRenou) [inline packages/pyolite-kernel/py/pyolite/pyolite/__init__.py:46]: This is my ticket to hell...

**Comment by martinRenou** [comment]
It's not quite right yet

**Comment by martinRenou** [comment]
Now working again :)

matplotlib

**Review by jtpio — APPROVED** [review]
Nice, thanks for the fix!

**Pull request merged**